From: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>
Subject: [PATCH] acpi_platform: Remove PNPC040

This is a PnP device managed by soc_button driver, so remove it from
the plafom ACPI devices list.

Signed-off-by: Faouaz TENOUTIT <faouaz.tenoutit@intel.com>

---
 drivers/acpi/acpi_platform.c |    1 -
 1 file changed, 1 deletion(-)

Index: kernel-starpeak-3.14/drivers/acpi/acpi_platform.c
===================================================================
--- kernel-starpeak-3.14.orig/drivers/acpi/acpi_platform.c	2014-04-07 12:17:23.840053312 +0200
+++ kernel-starpeak-3.14/drivers/acpi/acpi_platform.c	2014-04-07 12:17:43.524052557 +0200
@@ -36,7 +36,6 @@
 	{ "BCM2E1A" },
 	{ "BCM2E39" },
 	{ "BCM2E3D" },
-	{ "PNP0C40" }, /* GPIO buttons device */
 
 	/* Intel Smart Sound Technology */
 	{ "INT33C8" },
